other/cmr-exchange/ous-plugin/project.clj

```diff
@@ -9,6 +9,10 @@
     ;; this function is really only meant to be used by Dragon itself.
     (catch Exception _ "")))
 
+;; 33 == Yellow
+;; 34 == Blue
+;; 35 == Magenta
+;; [ns] λ=>
```

Comment:
I actually know these and can read what is going on, thought I would document it for others.